packages/camera/camera_android_camerax/README.md

```diff
@@ -19,21 +20,17 @@ dependency in addition to `camera`:
 dependencies:
   # ...along with your other dependencies
   camera: ^0.10.4
-  camera_android_camerax: ^0.5.0
+  camera_android_camerax: ^0.6.0
```

Comment:
This should land after https://github.com/flutter/packages/pull/6176 which will bump the plugin to version 0.6.0 and will make the changes below:

Reply:
Alternately, we could replace this whole section with something like "To use this plugin instead of `camera_android`, run `flutter pub add camera_android_camerax` in your project." That will give the latest version.